core/common/src/Localization.ts

```diff
@@ -9,7 +9,7 @@
 /** Options for Localization
  * @public
  */
-interface LocalizationOptions {
+export interface TranslationOptions {
   /** for interpolation values */
   [key: string]: any;
```

Comment:
you pretty much lose all intellisense with this

Reply:
What do mean? How so?

Reply:
i was talking about the `[key: string]: any;` 

but nvm, disregard this initial comment. i have seen places where adding a property to add anything like this just confuses the intellisense but after messing around in a ts playground to come up with an example it looks like it actually works fine now